Dockerfile

```diff
@@ -23,6 +23,24 @@ RUN cd /opt/android-ndk-tmp && mv ./android-ndk-r12 /opt/android-ndk
 RUN rm -rf /opt/android-ndk-tmp
 # add to PATH
 ENV PATH ${PATH}:${ANDROID_NDK_HOME}
+
+
+# ------------------------------------------------------
+# --- Android CMake
+
+# download
+RUN mkdir /opt/android-cmake-tmp
+RUN cd /opt/android-cmake-tmp && wget -q https://dl.google.com/android/repository/cmake-3.6.3155560-linux-x86_64.zip -O android-cmake.zip
+# uncompress
+RUN cd /opt/android-cmake-tmp && unzip -q android-cmake.zip -d android-cmake
+# move to it's final location
+RUN cd /opt/android-cmake-tmp && mv ./android-cmake ${ANDROID_HOME}/cmake
```

Comment:
thanks, I wonder why it's not part of the Android SDK then.. Well, I guess it's only required for NDK, but then why they chose to put it into the SDK home?..

Anyway, in case of Android it's not always "logical" how things work / where things are ¯\_(ツ)_/¯